source/django_package/domain/import_datas/parsing_config_file.py

```diff
@@ -0,0 +1,23 @@
+import yaml
+
+
+class ReadConfigFile:
```

Comment:
Inherit from object.
Add docstrings with description of what this class/method is doing and how.